pkg/controller/storageprofile-controller.go

```diff
@@ -216,7 +216,9 @@ func (r *StorageProfileReconciler) checkIncompleteProfiles() error {
 		return err
 	}
 	for _, profile := range storageProfileList.Items {
-		if isIncomplete(profile.Status.ClaimPropertySets) {
+		// We don't count explicitly unsupported provisioners as incomplete
+		_, found := storagecapabilities.UnsupportedProvisioners[*profile.Status.Provisioner]
```

Comment:
Can we ensure that whatever field we are dereferencing here is not nil before doing so. profile.Status is populated only after at least one reconcile happened, and I just want to ensure we don't get nil pointers.

Reply:
Makes sense.